script.js

```diff
@@ -0,0 +1,108 @@
+
+const taskInput = document.getElementById("todoInput");
+const addTaskBtn = document.getElementById("addButton"); //왜 안됌>?
+const pendingTasksList = document.getElementById("todosPendingList");
+const completedTasksList = document.getElementById("todosFinishedList");
+
+
+//create New todo format
+let createNewTask = function (todo) {
+    let newTodo = document.createElement("li");
+    let label = document.createElement("label");
+    let deleteBtn = document.createElement("button");
+
+    deleteBtn.className = "delete";
+    deleteBtn.innerHTML=`<img src="./img/bin.png", alt="bin", width=14px, height=14px>`;
+    label.innerText = todo;
+    label.className="label";
+
+    newTodo.appendChild(label);
+    newTodo.appendChild(deleteBtn);
+    console.log(newTodo)
+    return newTodo;
+}
+
+//add Task
+let addTask = function () {
+    console.log(" in add task");
+    let newTask = createNewTask(taskInput.value); //.value??
+    console.log(newTask)
+    console.log(pendingTasksList)
+    pendingTasksList.appendChild(newTask);
+    total=pendingTasksList.childElementCount
+    document.getElementById('pendingTasksNumber').innerHTML=total;
+    bindTaskEvents(newTask, completedTask);
+
+    taskInput.value = "";
+}
+//tasks number
+let tasksCount=function(){
```

Comment:
`countTasks`라고 동사형으로 통일해주는게 어떤 역할을 하는 함수명인지 더 잘 알 수 있기때문에 조금 더 명확한 네이밍 같습니당. [자바스크립트 명명규칙 참고](https://ui.toast.com/fe-guide/ko_CODING-CONVENSION/#%EC%BD%94%EB%94%A9-%EC%BB%A8%EB%B2%A4%EC%85%98)
```suggestion
let countTasks=function(){
```